src/pages/ReimbursementAccount/BankAccountManualStep.js

```diff
@@ -121,7 +121,7 @@ class BankAccountManualStep extends React.Component {
                                 </TextLink>
                             </View>
                         )}
-                        defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'acceptTerms', true)}
+                        defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'acceptTerms', false)}
```

Comment:
The only thing to note is that once a user checked it, they will see is selected by default next time the page will open as the form state is stored as a draft. This state will be cleared once the bank acc flow is completed.